manifests/b/Brave/BraveTalk/1.3.361.143/Brave.BraveTalk.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Brave.BraveTalk
+PackageVersion: 1.3.361.143
+InstallerType: exe
+InstallerSwitches:
+  Silent: silent
+  SilentWithProgress: -silent
+  Custom: ''

```

```suggestion
InstallModes:
- silentWithProgress
```
Unsure if this is allowed but Brave does it somehow